scripts/ngless-cwl/cwl/ngless-count.cwl

```diff
@@ -6,49 +6,48 @@
 cwlVersion: "cwl:v1.0"
 
 class: CommandLineTool
-baseCommand: ['./ngless-count.py']
+baseCommand: ['ngless-count.py']
+
+requirements:
+- $import: ngl-types.yml
 
 doc: |
   None
 
 inputs:
-  
   input:
-    type: str
-  
+    type: File
     doc: SAM/BAM/CRAM file to count reads on
     inputBinding:
-      prefix: --input 
+      prefix: --input
 
   output:
-    type: str
-  
+    type: string
     doc: Output file/path for results
     inputBinding:
-      prefix: --output 
+      prefix: --output
 
   features:
-    type: ["null", str]
+    type: ["null", string]
```

Comment:
FYI: you can use the shortcut `type: string?` (but it isn't wrong to keep what `argparse2tool` did here)